wiki/People/de.md

```diff
@@ -1,13 +1,10 @@
----
-outdated: true
-outdated_since: 7880a47c4e5df0bb9470cdd8edc48daa42e6f277
----
-
 # Menschen
```

Comment:
I'm thinking maybe this should be translated as `Personen` or `Leute`? I think both work better than `Menschen`. :thinking:

Reply:
Yeah, good point. I also thought about this myself. The main page uses `Personen`, so I'm going to stick to this. `Leute` sounds a tiny bit too casual in my opinion.

```suggestion
# Personen
```